src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -54,7 +53,7 @@
 #include "../world/MapAnimation.h"
 #include "../world/Park.h"
 #include "../world/Sprite.h"
-#include "../ride/Station.h"
+#include "S6Exporter.h"
```

Comment:
Strange, clang-format should have put this at the top as it's the 'main' include here.

Reply:
I sorted it manually.